setup-system.sh

```diff
@@ -43,8 +43,16 @@ function install_sublime_3() {
     sudo dpkg -i /tmp/sublime-text_build-3065_amd64.deb
 }
 
+# Download and install atom as best we know how
+function install_atom() {
+    echo "~ Downloading atom deb package"
+    sudo wget -P /tmp https://atom-installer.github.com/v1.5.3/atom-amd64.deb
```

Comment:
Sorry for the delay bro. Actually since this is our setup script and neither of us really care about Sublime, we could just remove it. But that's for another day. I'll merge this for now.